corehq/apps/registration/templates/registration/email/confirm_account_reminder.txt

```diff
@@ -0,0 +1,23 @@
+Dear {{ full_name }},
```

Comment:
the original confirm account email lacks translation too but in all practical terms how do we determine what language to send the confirm account email in?